python/ray/_raylet.pyx

```diff
@@ -1120,3 +1119,13 @@ cdef void async_retry_with_plasma_callback(shared_ptr[CRayObject] obj,
                 AsyncGetResponse(
                     plasma_fallback_id=ObjectID(object_id.Binary()),
                     result=None)))
+
+cdef void async_plasma_callback(CObjectID object_id,
+                                int64_t data_size,
+                                int64_t metadata_size) with gil:
+    message = []
+    message.append((ObjectID(object_id.Binary()), data_size, metadata_size))
```

Comment:
is there any reason not to use a list literal here? message = [(ObjectID(....)]

Reply:
No reason, I'll switch to a list literal.